docs/xml/collection-xmldata.xml

```diff
@@ -270,23 +270,28 @@
 						<literal>stock</literal> icons are loaded from stock. The icon name should never include any file-extension or path.
 					</para>
 					<para>
-						<literal>cached</literal> icons are loaded from the AppStream icon cache. The icon tag should contain the icon file name, including it's
+						<literal>cached</literal> icons are loaded from the AppStream icon cache. The icon tag should contain the icon file name, including its
 						extension. It must not contain a full or relative path to the icon file.
+						This icon type may have <literal>width</literal> and <literal>height</literal> properties.
+						If targeting a hi-DPI screen, this icon type may have a <literal>scale</literal> property.
```

Comment:
Could we briefly explain the semantic of "scale" here, even if it's obvious?
So something like `If targeting a hi-DPI screen, this icon type may have a <literal>scale</literal> property with an integer value > 1 to denote the icon's scale factor, e.g. <code>scale="2"</code> for 2x scaling.`
We could also refer to the Freedesktop Icon Theme Specification here, but that would likely be a bit too verbose.

Reply:
Done